buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -88,7 +104,7 @@ CustomInstallLib = patch_setuptools_command('install_lib')
             DESTINATION ${_package_install_destination}
             PATTERN "test" EXCLUDE )
 
-    if (APPLE AND "${pkg_name}" STREQUAL "mantidqt")
```

Comment:
I think the final 3 cases here should still be `pkg_name`?

Reply:
I've renamed `pkg_name` to `target_name`, so there's no more `pkg_name` in the namespace. I'll change it back to `pkg_name`. My original change was due to having a second argument, before realising it would be better to use `ARGN`

Reply:
Ah yes. I didn't read that properly. Thanks.